CellFabric/Cell_Fabric_FinFET__Mock/canvas.py

```diff
@@ -109,7 +127,7 @@ def _gen_abstract_MOS( self, x, y, x_cells, y_cells, fin_u, fin, finDummy, gate,
             for (pin, contact, track, m3route) in Routing:
                 self.addWire( self.m2, pin, pin, y*h+track, (min(contact), -1), (max(contact), 1))
                 if y_cells > 1:
-                   self.addWire( self.m3, None, None, m3route, (track, -1), (y*h+track, 1))
+                   self.addWire( self.m3, pin, pin, m3route, (track, -1), (y*h+track, 1))
```

Comment:
Might have to generalize pin names (probably done one level above this). For example, SA, SB may need to become M1:S, M2:S... Would be nice if we could see how the pin names get annotated in cadence pcells for example.

Reply:
Sorry I misspoke. pin name is fine at this level since we are talking about the top level connection. But we need to annotate device:pin at the V0 layer. I can take care of this after you merge this pull request as well.

Reply:
Create issue #168 for this. Marking as resolved for now.